tfhe/src/c_api/high_level_api/mod.rs

```diff
@@ -90,3 +90,14 @@ impl From<crate::FheTypes> for FheTypes {
         }
     }
 }
+
+#[test]
+fn fhe_types_enum_to_int_compatible2() {
+    use strum::IntoEnumIterator;
+
+    for rust_value in crate::FheTypes::iter() {
+        let c_value = FheTypes::from(rust_value);
+
+        assert_eq!(rust_value as i32, c_value as i32)
+    }
```

Comment:
needs to be in a #[cfg(test)] module or have the annotation

Reply:
huh !

Reply:
did you check this test runs with `make test_c_api_rs` ?

Reply:
Just did, it does